### ISSUE TITLE
NVDA resolved and unresolved PR, issue statistics

### ISSUE THREAD

**Issue by dingpengyu** [nvaccess/nvda#11259]
In order to help the code reviewers to better merge the code,
 this question is created here.
pull request:
There are currently a total of 81 [pull request](https://github.com/nvaccess/nvda/pulls) in NVDA waiting to be merged.
Of which have been merged in total1347.
Issues:
There are currently 2090 Issues waiting to be resolved.
In total, there are 7728 closed issues.
ps:
The upcoming NVDA2020.2 solves a total of 96 problems, and there are two problems waiting to be solved.
Hope this question can help code reviewers.
This Issues will be updated synchronously with [pr request](https://github.com/nvaccess/nvda/pulls) and [Issues](https://github.com/nvaccess/nvda/issues).
thanks

**Comment by lukaszgo1** [comment]
I'm not sure this has any advantages over looking at these statistics from GitHub's web interface.  @dingpengyu What improvements this gives for a reviewer in your opinion?

**Comment by dingpengyu** [comment]
I think it would be more convenient to make a unified Issues and pull request to organize code reviewers to query.

**Comment by feerrenrut** [comment]
You can also look at: https://github.com/nvaccess/nvda/pulse/monthly
You should be able to continue to comment on this even if I close it, I would prefer it closed since it is not an issue with NVDA.
